qiskit/pulse/commands/acquire.py

```diff
@@ -18,14 +18,11 @@
 
 from typing import Optional, Union, List
 
-from ..channels import MemorySlot, RegisterSlot, AcquireChannel
-from ..exceptions import PulseError
-
+from qiskit.pulse.channels import AcquireChannel, MemorySlot, RegisterSlot
+from qiskit.pulse.exceptions import PulseError
+from qiskit.pulse.instructions import Acquire, Instruction
 # pylint: disable=unused-import
 
```

Comment:
These files are being deleted soon, why make trivial changes?

Reply:
Reverted in bd508f772